recipe/meta.yaml

```diff
@@ -15,16 +15,14 @@ source:
   sha256: fb458ed205a5c3ed317568762e649eb8e87fd4d0118babfa46069e66154c70dd
 
 build:
-  merge_build_host: True  # [win]
```

Comment:
I'm not sure if this should be removed.

Reply:
It's a generic, so I assume this won't make any difference.